src/main/resources/fine_tuning/models.yaml

```diff
@@ -102,24 +122,26 @@ models:
         lower: 0
         upper: 0.5
         pace: 0.1
-      b:
-        type: float
-        lower: 0
-        upper: 0.5
-        pace: 0.05
-      k1:
-        type: float
-        lower: 0.9
-        upper: 0.9
-        pace: 0.1
     expected:
       robust04:
-        map: 0.2532
+        map:
+          best_avg: 0.0
+          oracles_per_topic: 0.0
+          2-fold: 0.0
+          5-fold: 0.0
+        P_20:
+          best_avg: 0.0
+          oracles_per_topic: 0.0
+          2-fold: 0.0
+          5-fold: 0.0
+        ndcg20:
+          best_avg: 0.0
+          oracles_per_topic: 0.0
+          2-fold: 0.0
+          5-fold: 0.0
   rm3:
```

Comment:
For `rm3` and `bm25+rm3`, `originalQueryWeight` can be `1.0`. I think we should have that. But then all combinations of other `rm3` parameters will be searched for `originalQueryWeight=1.0`. Then the whole program wastes some time on those? Can we get around it such as support having multiple parameter ranges? Not necessary needs to be changed in this PR but I think we probably should avoid that?